msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -372,8 +372,8 @@ items:
 
         :::image type="content" source="~/assets/images/sbs-notification-bot/provision-cloud.png" alt-text="Screenshot showing the provisioning commands":::
 
-    1. Select an existing or create new resource group.
-    
+    1. Select an existing or create new resource group. For more information, see [Create resource group](/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups).
```

Comment:
Select an existing what? Mention resource group. The sentence doesn't read well.

Reply:
Rectified.